plugins/kubernetes/app/models/kubernetes/role_verifier.rb

```diff
@@ -0,0 +1,106 @@
+module Kubernetes
+  class RoleVerifier
+    DEPLOYISH = ['Deployment', 'DaemonSet'].freeze
+
+    def initialize(role)
```

Comment:
Not clear what `role` is supposed to be here. I would've assumed a `Kubernetes::Role` object.  But it's actually supposed to be a string, right?